src/Common/src/Interop/Unix/libcrypto/Interop.Rsa.cs

```diff
@@ -83,6 +83,16 @@ internal static unsafe RSAParameters ExportRsaParameters(SafeRsaHandle key, bool
                     rsaParameters.Q = ExtractBignum(rsaStructure->q, halfModulus);
                     rsaParameters.DQ = ExtractBignum(rsaStructure->dmq1, halfModulus);
                     rsaParameters.InverseQ = ExtractBignum(rsaStructure->iqmp, halfModulus);
+
+                    if (rsaParameters.D == null ||
+                        rsaParameters.P == null ||
+                        rsaParameters.DP == null ||
+                        rsaParameters.Q == null ||
+                        rsaParameters.DQ == null ||
+                        rsaParameters.InverseQ == null)
```

Comment:
I have a vague recollection of seeing a set of checks like this elsewhere.  If I'm misremembering, ignore me.  If not, should we consolidate into a helper?